content/efcore/getting-started/samples/configuring-deletion/default-deletion-strategy/Program.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var reader = context.Readers.Include(r => r.Address).First();
+            var readers = context.Readers.First();
+            Console.WriteLine("{0} {1}", reader.FirstName, reader.LastName);
```

Comment:
Let's just remove the reader and address output. The output about whether the address has been deleted or not should be enough